### DISCUSSION TITLE
Feature: A CLI option to tell that `pnpm up --latest` should not change the major version

### DISCUSSION THREAD

**Discussion by babakfp** [pnpm/pnpm#8834] [Ideas]
This is my config:

```
save-exact=true
engine-strict=true
lockfile=false
```

When I do `pnpm up --latest`, it updates the major version too (as expected).

It seems like `pnpm up` doesn't do anything.

Something like this looks good to me:

- `pnpm up`: update only patch
- `pnpm up --latest-no-major`: update patch and minor but not major.

**Comment by zkochan** [comment]
It doesn't do anything because you use exact versions instead of version ranges in your `package.json`.

**Reply by babakfp** [reply]
I didn't want to use `^`/`~` and lock file, but ok.

**Reply by zkochan** [reply]
Well, your dependencies will still use ranges, so a lockfile is still needed.
